apps/sms/index.html

```diff
@@ -62,9 +62,6 @@
     -->
     <!-- Shared code -->
     <script defer src="shared/js/l10n.js"></script>
-    <script defer src="shared/js/l10n_date.js"></script>
-    <!-- Should be removed while gecko support navigator.mozHour12 API -->
-    <script defer src="shared/js/date_time_helper.js"></script>
```

Comment:
unrelated: can you also do the change in views/inbox/index.html ? It's not used yet but you can try to load it in Firefox in a DEBUG=1 DESKTOP=0 profile.